botocore/serialize.py

```diff
@@ -453,6 +455,11 @@ def _serialize_payload(self, partitioned, parameters,
             serialized['body'] = self._serialize_body_params(
                 partitioned['body_kwargs'], shape)
 
+    def _encode_payload(self, body):
+        if isinstance(body, six.text_type):
+            return body.encode(self.DEFAULT_ENCODING)
```

Comment:
I think we should probably update these to all use `self.DEFAULT_ENCODING`.  At least that way it's configurable if someone needs to modify this.